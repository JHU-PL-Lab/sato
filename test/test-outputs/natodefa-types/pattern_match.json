[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [],
          "err_match_val": "{}",
          "err_match_expected": "List",
          "err_match_actual": "Record {}"
        }
      ],
      [
        "Error_match",
        {
          "err_match_aliases": [],
          "err_match_val": "{}",
          "err_match_expected": "List",
          "err_match_actual": "Record {}"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "match {} with | hd :: tl -> tl | [] -> []\nend",
    "err_steps": 110
  }
]