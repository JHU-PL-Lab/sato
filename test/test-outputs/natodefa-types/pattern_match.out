Type errors for:
- Input sequence : []
- Found at expr  : match {} with | hd :: tl -> tl | [] -> []
end
--------------------
* Value    : {}
* Expected : List
* Actual   : Record {}
--------------------
* Value    : {}
* Expected : List
* Actual   : Record {}
2 errors found. No further control flows exist.
